ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Hey team! 
+
+I was using your awesome geocoding engine when I noticed something interesting.
```

Comment:
I thought it would come across as setting a friendlier tone in the rest of the message. I don't mind being a bit too cheerful here. 🎉